theme.json

```diff
@@ -0,0 +1,403 @@
+      "space.075": "0.375rem",
+      "space.100": "0.5rem",
+      "space.150": "0.75rem",
+      "space.200": "1rem",
+      "space.300": "1.5rem",
+      "space.400": "2rem",
+      "space.600": "3rem",
+      "space.800": "4rem",
+      "space.900": "4.5rem", 
+      "space.1200": "6rem"
+    },
+    "responsive": {
+      "space.2xs": {
+        "default": "0.25rem",
+        "sm": "0.25rem",
+        "md": "0.25rem"
+      },
+      "space.xs": {
+        "default": "0.375rem",
+        "sm": "0.5rem",
+        "md": "0.5rem"
+      },
+      "space.sm": {
+        "default": "0.5rem",
+        "sm": "0.75rem",
+        "md": "1rem"
+      },
+      "space.md": {
+        "default": "1rem",
+        "sm": "1.5rem",
+        "md": "2rem"
+      },
+      "space.lg": {
+        "default": "1.5rem",
+        "sm": "2rem",
+        "md": "3rem"
+      },
+      "space.xl": {
+        "default": "2rem",
+        "sm": "3rem",
+        "md": "4rem"
+      },
+      "space.2xl": {
+        "default": "3rem",
+        "sm": "4.5rem",
+        "md": "6rem"
+      }
+    }
+  },
+  "font": {
+    "size": {
+      "f-3": "clamp(0.69rem, calc(0.06vw + 0.68rem), 0.75rem)",
+      "f-2": "clamp(0.81rem, calc(0.06vw + 0.80rem), 0.88rem)",
+      "f-1": "clamp(0.88rem, calc(0.19vw + 0.84rem), 1.06rem)",
+      "f0": "clamp(1.00rem, calc(0.25vw + 0.95rem), 1.25rem)",
+      "f1": "clamp(1.13rem, calc(0.38vw + 1.05rem), 1.50rem)",
+      "f2": "clamp(1.25rem, calc(0.56vw + 1.14rem), 1.81rem)",
+      "f3": "clamp(1.44rem, calc(0.75vw + 1.29rem), 2.19rem)",
+      "f4": "clamp(1.63rem, calc(0.94vw + 1.44rem), 2.56rem)",
+      "f5": "clamp(1.81rem, calc(1.31vw + 1.55rem), 3.13rem)",
+      "f6": "clamp(2.00rem, calc(1.75vw + 1.65rem), 3.75rem)"
+    },
+    "family": {
+      "brand": "\"Wellcome\", sans-serif",
+      "sans": "\"Inter\", sans-serif",
+      "mono": "\"IBM Plex Mono\", “Courier New”, Courier, monospace"
+    },
+    "weight": {
+      "light": "300",
+      "regular": "400",
+      "medium": "500",
+      "semibold": "600",
+      "bold": "700"
+    }
+  },
+  "letter-spacing": {
+    "tight": "-0.03125rem",
+    "comfortable": "0.0125rem",
+    "spacious": "0.025rem"
+  },
+  "line-height": {
+    "sm": "100%",
+    "md": "130%",
+    "lg": "150%"
+  },
+  "text-transform": {
+    "uppercase": "uppercase"
+  },
+  "opacity": {
+    "transparent": "0",
+    "25": "0.25",
+    "50": "0.50",
+    "75": "0.75",
+    "100": "1"
+  },
+  "grid": {
+    "columns": {
+      "default": "6",
+      "sm": "12"
+    },
+    "gutter": {
+      "default": "0.75rem",
+      "sm": "1.5rem",
+      "lg": "3rem"
+    },
+    "margin": {
+      "default": "1rem",
+      "sm": "1.25rem",
+      "lg": "4.5rem"
+    }
+  },
+  "border-radius": {
+    "none": "0",
+    "xs": "0.125rem",
+    "sm": "0.25rem",
+    "md": "0.5rem",
+    "lg": "1rem",
+    "full": "9999px"
+  },
+  "color": {
+    "white": "#FFFFFF",
+    "black": "#000000",
+    "annotations": {
```

Comment:
I think that _annotations_ are the colours used only by designers in Figma, so we probably don’t need them in the design system. @dana-saur could you please confirm?